platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -24,7 +24,7 @@ NS_ASSUME_NONNULL_BEGIN
  alternatively observe KVO change notifications to the pack’s `progress` key
  path.
  */
-extern NSString * const MGLOfflinePackProgressChangedNotification;
+extern NSNotificationName const MGLOfflinePackProgressChangedNotification;
```

Comment:
Swift 3 bridges this symbol as `NSNotification.Name.MGLOfflinePackProgressChanged`, which is consistent with how AppKit and UIKit string enumeration values are being bridged.